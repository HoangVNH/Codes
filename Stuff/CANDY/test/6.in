10
3
VVV
8
XXXDDDXX
4
VVVX
3
VVX
3
VVV
1
V
4
VVVX
4
VVVV
6
VVXXXV
6
XXXXXX
