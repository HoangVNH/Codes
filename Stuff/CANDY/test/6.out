0
7
3
2
0
0
3
0
5
0
